.github/workflows/trivy.yml

```
@@ -0,0 +1,46 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Code Scanning
+
+on:
+  push:
+    branches: [ "develop", "main-DEPRECATED", "release/*" ]
```

```suggestion
    branches: [ "develop", "release/*" ]
```
Removing the main from the push targets